project.css

```diff
@@ -0,0 +1,171 @@
+.project-container {
+  width: 90%;
+  min-height: 90vh;
+  margin: 24px;
+  border: 2px solid #fff;
+  z-index: 2;
+  top: 20px;
+  background: rgba(255, 255, 255, 0.16);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.hidden {
+  display: none;
+}
+
+.active {
+  display: block;
+}
+
+.close-button {
+  align-self: flex-end;
+  margin: 20px;
+}
+
+.heading {
+  background: transparent;
+  font-size: 24px;
+  font-family: Roboto, sans-serif;
+  clear: right;
+  color: #fff;
+  padding-bottom: 24px;
+  align-self: flex-start;
+  padding-left: 47px;
+}
+
+.background-div {
+  width: 100%;
+  height: 100vh;
+  z-index: 2;
+  position: fixed;
+  top: 0;
+  bottom: 30px;
+  background: url("./assets/images/Content.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-color: #191765;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
+
+.description {
+  background-color: transparent;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  width: 279px;
+  padding-top: 24px;
+  padding-bottom: 24px;
+}
+
+.list {
+  background: transparent;
+  display: flex;
+  list-style-type: none;
+  justify-content: space-evenly;
+}
+
+.list > li {
+  color: #fff;
+  padding: 10px;
+  border-left: 1px solid #81818c;
+}
+
+.list > li:last-child {
+  border-right: 1px solid #81818c;
+}
+
+.links {
+  color: #fff;
+  display: block;
+  width: 150px;
+  padding: 15px 0 15px 0;
+  text-align: center;
+  border: 1px solid #fff;
+  background-color: #7d73fc;
+  margin: 17px 0 10px 0;
+  text-decoration: none;
+}
+
+.blur {
+  filter: blur(10px);
+}
+
+.scroll {
+  overflow: scroll;
+}
+
+.visible {
+  overflow: visible;
+}
+
+.noScroll {
+  overflow: hidden;
+}
+
+@media (min-width: 576px) {
+  .project-container {
+    margin-left: 5%;
+    overflow: visible;
+  }
+
+  .projectImage {
+    width: 80vw;
+  }
+
+  .description {
+    width: 50vw;
+    font-family: Roboto, sans-serif;
+    font-size: 20px;
+    font-weight: normal;
+    line-height: 30px;
+    text-align: center;
+  }
+
+  .button-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    gap: 30px;
+    align-items: center;
+    margin-bottom: 50px;
+    margin-top: 20px;
+  }
+
```

Comment:
- [ ] Kindly add these icons to the `See live` and `See source` buttons of your popup window :arrow_down: 

Your design: 

![b1](https://user-images.githubusercontent.com/57562869/152427765-e579a11b-5cae-4cc4-8ccf-055e49bd3261.png)


Figma:

![b1](https://user-images.githubusercontent.com/57562869/152427815-b7e30228-947b-4af9-b468-4f574aec1b0a.png)